packages/@react-spectrum/autocomplete/stories/SearchAutocomplete.stories.tsx

```diff
@@ -297,3 +299,48 @@ export const WithAvatars: SearchAutocompleteStory = {
     </SearchAutocomplete>
   )
 };
+
```

Comment:
added an async example of SearchAutocomplete so we could test the "No results" but if we wanna remove this lemme know

Reply:
It's not showing No results for me

Reply:
Oh interesting...it shows it for me. If you type in a string like "ajdkfla" does it say it?

Reply:
o, haha, I just opened it and expected it to not have any results, I can do it that way